.claude/commands/babysit-pr.md

```
@@ -98,13 +98,13 @@ Use the severity triage from the `bot-triage` skill:
 5. Reply to each fixed thread:
 
 ```bash
-./scripts/core/gh-review-helper.sh reply PR_NUMBER COMMENT_ID 'Fixed in COMMIT_SHA: description.'
+agentive review-helper reply PR_NUMBER COMMENT_ID 'Fixed in COMMIT_SHA: description.'
 ```
 
 6. Resolve all addressed threads:
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/1/1

(MD029, ol-prefix)